src/main/resources/plugin.yml

```diff
@@ -17,7 +17,12 @@ commands:
   movementtracersetting:
     description: Configure movement tracer settings
     usage: /movementtracersetting [HIGH | LOW | OFF]
+  togglekillcred:
+    description: Configure if you want to obfuscate your kill credit or not. This is a toggleable command.
 permissions:
   movecraft.combat.bypass:
     description: Allows you to bypass combat release punishments.
+    default: false
+  movecraft.command.hidekillcred:
+    description: Allows you to obfuscate you from showing up in the kill credits.
```

Comment:
Can we rename this feature to obfuscateKillCredit instead?  I think it better describes the functionality.